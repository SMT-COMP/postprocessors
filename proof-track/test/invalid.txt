0.00/0.00	; run solver
1.27/0.62	; checking proof
1.27/1.04	invalid
1.27/1.12	; done
1.27/1.12	EOF
