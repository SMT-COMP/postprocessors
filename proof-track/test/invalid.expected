starexec-result=starexec-unknown
reason=invalid
